src/contexts/CardView/CardView.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+  createContext,
+  FunctionComponent,
+  useCallback,
+  useContext,
+  useState,
+} from "react";
+import { PackageCardType } from "../../components/PackageCard";
+import { Controls } from "./Controls";
+
+const CardViewContext = createContext({
+  cardView: PackageCardType.Wide,
+  // eslint-disable-next-line react/display-name
+  CardViewControls: Controls as FunctionComponent,
+  setCardView: (() => {}) as (v: PackageCardType) => void,
+});
+
+export const useCardView = () => useContext(CardViewContext);
+
+export const CardViewProvider: FunctionComponent = ({ children }) => {
```

Comment:
This is setup as a context rather than a state within respective lists as it allows the user's selection to persist throughout the session and apply to any list view